extensions/amp-list/0.1/amp-list.js

```diff
@@ -851,8 +852,6 @@ export class AmpList extends AMP.BaseElement {
           this.unlistenLoadMore_ = listen(
               this.loadMoreService_.getLoadMoreButtonClickable(),
               'click', () => this.loadMoreCallback_());
-        }).then(() => {
```

Comment:
This is no longer necessary because the `attemptToFit` function has been refactored to branch if load-more is active. The logic has been moved there to make space for the `load-more-button` as follows: 

```

  /**
   * @param {number} targetHeight
   * @param {number} height
   * @private
   */
  attemptToFitLoadMoreElements_(targetHeight, height) {
    const loadMoreHeight = this.loadMoreService_
        .getLoadMoreButton()./*OK*/offsetHeight;
    if (targetHeight + loadMoreHeight > height) {
      this.attemptChangeHeight(targetHeight + loadMoreHeight)
          .then(() => {
            this.resizeFailed_ = false;
            // If there were not enough items to fill the list, consider
            // automatically loading more if load-more="auto" is enabled
            if (this.element.getAttribute('load-more') === 'auto') {
              this.maybeLoadMoreItems_();
            }
            setStyles(dev().assertElement(this.container_), {
              'max-height': '',
            });
          })
          .catch(() => {
            this.resizeFailed_ = true;
            this.adjustContainerForLoadMoreButton_();
          });
    }
  }

```